libs/agno/agno/models/xai/xai.py

```diff
@@ -36,117 +24,3 @@ class xAI(OpenAILike):
 
     api_key: Optional[str] = getenv("XAI_API_KEY")
     base_url: Optional[str] = "https://api.x.ai/v1"
-
-    def invoke_stream(self, messages: List[Message]) -> Iterator[ChatCompletionChunk]:
```

Comment:
oh this is because X release API support right. cool

Reply:
Yeah they broke it, then fixed it